images/macos/provision/utils/utils.sh

```diff
@@ -171,3 +171,28 @@ configure_user_tccdb () {
     local sqlQuery="INSERT OR IGNORE INTO access VALUES($values);"
     sqlite3 "$dbPath" "$sqlQuery"
 }
+
+get_github_package_download_url() {
+    local REPO_OWNER=$1
+    local REPO_NAME=$2
+    local FILTER=$3
+    local VERSION=$4
+    local API_PAT=$5
+    local SEARCH_IN_COUNT="100"
+
+    if [ $API_PAT ]; then
+        json=$(curl -H "Authorization: token $API_PAT" -s "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/releases?per_page=${SEARCH_IN_COUNT}")
+    else
+        json=$(curl -s "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/releases?per_page=${SEARCH_IN_COUNT}")
+    fi    
```

Comment:
Maybe get rid of `else`?
```suggestion
    if [ $API_PAT ]; then
        AuthString="-H 'Authorization: token $API_PAT'"
    fi

    json=$(curl $AuthString -s "https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/releases?per_page=${SEARCH_IN_COUNT}")
```